### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by Mic92** [kisli/vmime#274]
Could we get a new release? The last one was 5 years ago. Thanks

**Comment by Thermi** [comment]
Yeah that'd be really necessary now so package maintainers can upgrade well

**Comment by arrowd** [comment]
The last release does not build with the upcoming CMake 4.5, so we'll probably have to remove this library from FreeBSD packages.
